index.js

```diff
@@ -0,0 +1,19 @@
+const userLogin = document.querySelector("#user-login");
+
+const setName = (name) => {
+    if (name) {
+        userLogin.innerHTML = `Hello, ${name}!`;    
+    }
+}
```

Comment:
Is arrow function supported on all the browsers? Preferable to use ES5 syntax here?

Reply:
IE and Opera mini don't support arrow functions. Fixed.

Reply:
Not fixed for `fetchData`